Resources/Prototypes/Catalog/Fills/Lockers/medical.yml

```diff
@@ -35,41 +35,40 @@
         amount: 2
       - id: Gauze
 
-
 - type: entity
   id: LockerMedicalFilled
   suffix: Filled
   parent: LockerMedical
   components:
   - type: StorageFill
     contents:
-      - id: HandheldHealthAnalyzer
-        prob: 0.6
-      - id: ClothingHandsGlovesLatex
-      - id: ClothingHeadsetMedical
-     # - id: ClothingEyesHudMedical #Removed until working properly
-      - id: ClothingBeltMedical
-      - id: ClothingHeadHatSurgcapGreen
-        prob: 0.1
-        orGroup: Surgcaps
-      - id: ClothingHeadHatSurgcapPurple
-        prob: 0.05
-        orGroup: Surgcaps
-      - id: ClothingHeadHatSurgcapBlue
-        prob: 0.90
-        orGroup: Surgcaps
-      - id: UniformScrubsColorBlue
-        prob: 0.5
-        orGroup: Surgshrubs
-      - id: UniformScrubsColorGreen
-        prob: 0.1
-        orGroup: Surgshrubs
-      - id: UniformScrubsColorPurple
-        prob: 0.05
-        orGroup: Surgshrubs
-      - id: ClothingMaskSterile
-      - id: HandLabeler # Frontier
-      - id: ClothingBeltMedical # Frontier
+#      - id: HandheldHealthAnalyzer
+#        prob: 0.6
+#      - id: ClothingHandsGlovesLatex
+#      - id: ClothingHeadsetMedical # Frontier
+      - id: ClothingEyesHudMedical # Frontier
+#      - id: ClothingBeltMedical
+#      - id: ClothingHeadHatSurgcapGreen
+#        prob: 0.1
+#        orGroup: Surgcaps
+#      - id: ClothingHeadHatSurgcapPurple
+#        prob: 0.05
+#        orGroup: Surgcaps
+#      - id: ClothingHeadHatSurgcapBlue
+#        prob: 0.90
+#        orGroup: Surgcaps
+#      - id: UniformScrubsColorBlue
+#        prob: 0.5
+#        orGroup: Surgshrubs
+#      - id: UniformScrubsColorGreen
+#        prob: 0.1
+#        orGroup: Surgshrubs
+#      - id: UniformScrubsColorPurple
+#        prob: 0.05
+#        orGroup: Surgshrubs
+#      - id: ClothingMaskSterile
+#      - id: ClothingBeltMedical # Frontier
+      - id: ClothingBeltParamedicFilled # Frontier
```

Comment:
why? this isnt a paramedic locker, and the locker still needs some regular meds please

Reply:
Used wrong belt by mistake.

Reply:
its still paramedic belt

Reply:
Looking at the new file its seem to have the medical one.